webservice/assets/peoples.js

```diff
@@ -146,5 +146,14 @@ export function getPeopleList() {
 |         2022 Academician at Daskom1337         |
 +------------------------------------------------+`,
     }
-  ];
+  
+    {
+      Avatar: "RafiM.html",
+      Handle: "Bono Black Boy",
+      Realname: "Rafi Mahfuz Setyagraha",
+      Description: `
+  +------------------------------------------------+
+  |           I am inside your walls               |
+  +------------------------------------------------+`,
+  }];
 }
```

Comment:
These lines is incorrect Rafi, you have to change it to something like this one
```
    ,{
        Avatar: "RafiM.html",
        Handle: "Bono Black Boy",
        Realname: "Rafi Mahfuz Setyagraha",
        Description: `
+------------------------------------------------+
|           I am inside your walls               |
+------------------------------------------------+`,
    }
  ];
}
```
You missed that `,` before the curly brace and also dont forget to put the `];` in the new line to keep things clean 👍

Reply:
Okeey bang